doc/man/scons.xml

```diff
@@ -6703,23 +6706,66 @@ to enclose a Python expression to be evaluated.
 See <xref linkend='python_code_substitution'/> below
 for a description.</para>
 
-<para>A variable name
-may also be a Python function
-associated with a
-&consvar; in the environment.
-The function should
-accept four arguments:
+<para>The special pseudo-variables
```

Comment:
escape characters?

Reply:
?

Reply:
special escape characters? vs pseudo-variables?

Reply:
Well, I was using the wording already there.  If you think "special escape" is a better term, I've got no problem.

Reply:
updated